CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+CU_SET_PATH("CMAKE_MOD_BUFFCOMMAND_DIR" "${CMAKE_CURRENT_LIST_DIR}")
```

Comment:
```diff
-CU_SET_PATH("CMAKE_MOD_BUFFCOMMAND_DIR" "${CMAKE_CURRENT_LIST_DIR}")
+CU_SET_PATH("MODULE_CONFIG_FILE_LIST" "${CMAKE_CURRENT_LIST_DIR}")
```

This is needed, right?

Reply:
Okay, i got it now, from that commit

Reply:
https://github.com/azerothcore/azerothcore-wotlk/blob/d29a1a330018b89ee608aa27e500068f5310ed2a/src/cmake/ac_macros.cmake#L26